README.md

```diff
@@ -10,35 +10,98 @@ A dataset may have hundreds of fields. The dataset owner
 must [curate a list](https://github.com/DataBiosphere/data-explorer/blob/master/dataset_config/1000_genomes/ui.json)
 of the most important fields (age, gender, etc).
 
-A dataset has a notion of `primary_key`. For a dataset that tracks 1000
-participants, `primary_key` could be `participant_id`. For a dataset that
-contains 1000 samples from a single person, `primary_key` could be `sample_id`.
-
-`primary_key` is used to tie information together from different sources.
-Say there are facets for age and weight; age and weight are
-stored in separate BigQuery tables; and `primary_key` is `participant_id`.
-First, age table is indexed. An Elasticsearch document is created for each
-`participant_id` with document id = `participant_id`. A document would look
-like:
+A dataset has a notion of `participant_id` and (optional) `sample_id`. Currently
```

Comment:
Do you mind editing the first comment on this PR? Super_Population_Description should only be top-level, and not inside samples doc, right?